tests/e2e/e2e-tests/block_volume_attach.go

```diff
@@ -100,7 +100,11 @@ var _ = framework.KubeDescribe("[Feature:Block_Volume_Attach_E2E]", func() {
 				Expect(err).NotTo(HaveOccurred())
 				attachStatus, err := getAttchStatus()
 				Expect(err).NotTo(HaveOccurred())
-				Expect(pv.ObjectMeta.Annotations["ibm.io/dm"]).To(ContainElement("/dev/dm-"))
+                                devicePath := pv.ObjectMeta.Annotations["ibm.io/dm"]
```

Comment:
Why did you remove the earlier expect and add this `if` check? I think earlier one can be retained unless you have a specific reason.

Reply:
The  retunred value from  the devicePath := pv.ObjectMeta.Annotations["ibm.io/dm"] is string but the COntianElement expects the different type thats y changed